### PR TITLE
chore: release 0.26.0

### PR DESCRIPTION
**Pull request by oleobal**

### CODE REVIEW

File: docs/source/additional/release.rst [hunk 1]
```diff
@@ -28,24 +28,50 @@ This is an overview of the main changes, please have a look at the changelog of
...
 
-Substra 0.25.0 - 2023-02-17
-^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Substra 0.26.0 --- 2023-04-03
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Improve backend performance when handling large amounts of compute plans and tasks. This will result in faster front-end pages.
+- Rename Algo to Function.
+- SSO Login
+- Add a task duration breakdown for every task in the front-end:
```
Comment (RomainGoussault) [inline docs/source/additional/release.rst:37]:
```suggestion
- [Experimental] Add a task duration breakdown for every task in the front-end. Note that this is an experimental feature and only works on the current backend you are logged into.
```

File: docs/source/conf.py [hunk 2]
```diff
@@ -29,9 +29,9 @@
 TMP_FOLDER = Path(__file__).parents[2] / "tmp"
 
 if os.environ.get("READTHEDOCS_VERSION_TYPE") == "tag":
-    SUBSTRA_VERSION = "0.42.0"
+    SUBSTRA_VERSION = "0.43.0rc1"
```
Comment (SdgJlbl) [inline docs/source/conf.py:32]: do we want to keep the rc here?

File: .gitignore [hunk 3]
```diff
@@ -151,6 +151,7 @@ docs/source/substrafl_doc/examples
 # Data folder when run locally
 **/data_iris
 **/data_mnist
+**/data
```
Comment (SdgJlbl) [inline .gitignore:154]: where is this used?

File: docs/source/conf.py (hunk 2 above)
Comment (oleobal) [reply docs/source/conf.py:32]: Good catch

File: .gitignore (hunk 3 above)
Comment (oleobal) [reply .gitignore:154]: I'm not sure, this is a Thibault commit

**Review by SdgJlbl — APPROVED** [review]

**Pull request merged**